docs-src/content/using-cubic-sdk/streaming.md

```diff
@@ -111,6 +111,73 @@ for resp in client.StreamingRecognize(cfg, audio):
             print("\r{0}".format(result.alternatives[0].transcript), end="\n")
 {{< /tab >}}
 
+{{< tab "C++" "c++" >}}
+#include "cubic_client.h"
+
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <thread>
+
+namespace CubicPB = cobaltspeech::cubic;
+const std::string serverAddress = "localhost:2727";
+const std::string filename = "test.raw";
+
+// This client demonstrates using synchronous recognition.
+int main(int argc, char *argv[]) {
+    // Create the client (note this is an insecure connection,
+    // which is not recommended for production).
+    CubicClient client(serverAddress);
+
+    // Get the list of available models
+    std::vector<CubicModel> models = client.listModels();
+    std::cout << "Available Models:" << std::endl;
+    for (const CubicModel &m : models) {
+        std::cout << "ID = " << m.id() << ", Name = " << m.name() << std::endl;
+    }
+
+    // Use the first model to set up the recognition config
+    auto modelID = models[0].id();
+    CubicPB::RecognitionConfig cfg;
+    cfg.set_model_id(modelID);
+    cfg.set_audio_encoding(CubicPB::RecognitionConfig::RAW_LINEAR16);
```

Comment:
Do we want to keep things consistent across the languages for this example?  I'm not opposed to doing one some examples in WAV and some examples in RAW_LINEAR16, but it might be nice to keep things consistent between languages for any given example.

Reply:
This could be a separate PR if needed.

Reply:
We do want to be consistent. I thought I had done that, but it looks like I used raw for this example and wav for the synchronous recognition example, which is backwards from all the other examples. I'll update and fix.